Essentials/src/com/earth2me/essentials/messaging/SimpleMessageRecipient.java

```diff
@@ -147,6 +150,19 @@ public MessageResponse onReceiveMessage(IMessageRecipient sender, String message
         // Display the formatted message to this recipient.
         sendMessage(tl("msgFormat", sender.getDisplayName(), tl("meRecipient"), message));
 
+        // Play a "ding" sound when receiving a message
+        if (user != null && !user.isIgnoreMsgSound()) {
+            Sound dingSound;
+
+            if (VersionUtil.getServerBukkitVersion().isHigherThanOrEqualTo(VersionUtil.v1_9_R01)) {
+                dingSound = Sound.ENTITY_EXPERIENCE_ORB_PICKUP;
+            } else {
+                dingSound = Sound.valueOf("ORB_PICKUP");
```

Comment:
You can literally make sound configurable and use `bool ? : `

Reply:
True, just didn't to make it easier to read